src/tracks/playabletrack/wavetrack/ui/SpectrumCache.cpp

```diff
@@ -573,7 +573,7 @@ bool WaveClipSpectrumCache::GetSpectrogram(const WaveClip &clip,
    // purposely offset the display 1/2 sample to the left (as compared
    // to waveform display) to properly center response of the FFT
    fillWhere(mSpecCache->where, numPixels, 0.5, correction,
-      t0, rate, samplesPerPixel);
+      t0, rate, 1.0, samplesPerPixel);
```

Comment:
Maybe you did this at a later commit -- but while I review "Make waveform display work again", I see that the `0.5` should be changed to `true`